include/guestbook_db.inc

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+* @file
+* Implementation of database access functions.
+*/
+
+/**
+ * Reading data from the database.
+ *
+ * @return
+ *   A data array containing complete information about each database entry.
+ */
+function guestbook_read_db($table = 'guestbook_drupal', $order = 'id', $direction = 'DESC', $limit = '10') {
+  $rows = array();
+  $query = db_select($table, 't')
+    ->extend('PagerDefault')
+    ->fields('t', array('id', 'uid', 'sid', 'date', 'message'))
+    ->orderBy($order, $direction)
+    ->limit($limit)
+    ->execute();
+  while ($value = $query->fetchAssoc()) {
+    $rows[] = [
+      $value['id'],      // Message id.
+      $value['uid'],     // User id.
+      $value['sid'],     // Session id.
+      $value['date'],    // Creation date of message.
+      $value['message'], // Message text.
+    ];
+  }
+  return $rows;
+}
+
+/**
+ * Save or edit message.
+ *
+ * @param $form_state
```

Comment:
after `@param $form_state` should follow only param description. If you would like to add the function description you should add it after function main name like here
`
/**
\* Save or edit message.
\*
\* ...description here ...
\*
\* @param ....

`